docs/splatalogue/splatalogue.rst

```diff
@@ -218,48 +218,37 @@ regular basis.  For example, if you want data returned preferentially in units
     ...     return table
     >>> trimmed_query(1*u.GHz,30*u.GHz,
     ...     chemical_name='(H2.*Formaldehyde)|( HDCO )',
-    ...     energy_max=50)[:10].pprint(max_width=100)
-    Species Chemical Name         QNs          Freq-GHz Meas Freq-GHz log10(Aij)   EU_K
-    ------- ------------- ------------------- --------- ------------- ---------- --------
-       HDCO  Formaldehyde       1(1,0)-1(1,1)        --     5.3461416   -8.31616 11.18301
-     H2C18O  Formaldehyde 1(1,0)-1(1,1),F=1-0 4.3887783            --    -9.0498 15.30206
-     H2C18O  Formaldehyde 1(1,0)-1(1,1),F=0-1 4.3887957            --   -8.57268 15.30206
-     H2C18O  Formaldehyde 1(1,0)-1(1,1),F=2-2 4.3887965            --   -8.69765 15.30206
-     H2C18O  Formaldehyde       1(1,0)-1(1,1)  4.388797            --   -8.57272 15.30206
-     H2C18O  Formaldehyde 1(1,0)-1(1,1),F=2-1 4.3888012            --   -9.17475 15.30206
-     H2C18O  Formaldehyde 1(1,0)-1(1,1),F=1-2 4.3888036            --    -8.9529 15.30206
-     H2C18O  Formaldehyde 1(1,0)-1(1,1),F=1-1 4.3888083            --    -9.1748 15.30206
-     H213CO  Formaldehyde       1(1,0)-1(1,1)        --     4.5930885   -8.51332 15.34693
-       H2CO  Formaldehyde 1(1,0)-1(1,1),F=1-2 4.8296665            --   -8.82819 15.39497
-
+    ...     energy_max=50)[:10].pprint(max_width=150)
+    species_id chemical_name             name                     resolved_QNs         orderedfreq   aij    upper_state_energy_K
+    ---------- ------------- ---------------------------- ---------------------------- ----------- -------- --------------------
+           109  Formaldehyde                         HDCO       1(  1, 0)-   1(  1, 1)    5346.142 -8.44112             11.18258
+           109  Formaldehyde                         HDCO           1( 1, 0)- 1( 1, 1)   5346.1616 -8.31295             11.18287
+           109  Formaldehyde                         HDCO              1(1,0) - 1(1,1)   5346.1416 -8.31616             11.18301
+           155  Formaldehyde H<sub>2</sub>C<sup>18</sup>O       1(  1, 0)-   1(  1, 1)    4388.797 -8.22052             15.30187
+           155  Formaldehyde H<sub>2</sub>C<sup>18</sup>O  1( 1, 0)- 1( 1, 1), F= 1- 0   4388.7783  -9.0498             15.30206
+           155  Formaldehyde H<sub>2</sub>C<sup>18</sup>O  1( 1, 0)- 1( 1, 1), F= 0- 1   4388.7957 -8.57268             15.30206
+           155  Formaldehyde H<sub>2</sub>C<sup>18</sup>O  1( 1, 0)- 1( 1, 1), F= 2- 2   4388.7965 -8.69765             15.30206
+           155  Formaldehyde H<sub>2</sub>C<sup>18</sup>O              1(1,0) - 1(1,1)    4388.797 -8.57272             15.30206
+           155  Formaldehyde H<sub>2</sub>C<sup>18</sup>O  1( 1, 0)- 1( 1, 1), F= 2- 1   4388.8012 -9.17475             15.30206
+           155  Formaldehyde H<sub>2</sub>C<sup>18</sup>O  1( 1, 0)- 1( 1, 1), F= 1- 2   4388.8036  -8.9529             15.30206
+
+There are utility functions in `astroquery.splatalogue.utils` that automate
```

Comment:
These are not in the public API, should we 1) put it there 2) maybe no advertise it?

Either case, the fix below gets rid of the RTD CI failure
```suggestion
There are utility functions in ``astroquery.splatalogue.utils`` that automate
```

Reply:
They kinda should be in the public API, but I don't want to make that change now - your workaround is better for the moment.